forge-gui/res/cardsfolder/upcoming/brotherhood_patriarch.txt

```diff
@@ -0,0 +1,9 @@
+Name:Brotherhood Patriarch
+ManaCost:3 B
+Types:Creature Human Assassin
+PT:4/1
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Card.Self | Execute$ TrigLoseLife | TriggerDescription$ When CARDNAME dies, each opponent loses 2 life and you gain 2 life.
+SVar:TrigLoseLife:DB$ LoseLife | Defined$ TriggeredCardOpponent | LifeAmount$ 2 | SubAbility$ DBGainLife
```

Comment:
weird defined

Reply:
I took that from Etched Familiar, maybe it needs to be changed there as well?

Reply:
yea just seems messy